app/src/main/java/com/kickstarter/ui/activities/HelpSettingsActivity.kt

```diff
@@ -0,0 +1,85 @@
+package com.kickstarter.ui.activities
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.text.TextUtils
+import com.kickstarter.R
+import com.kickstarter.extensions.startActivityWithSlideLeftTransition
+import com.kickstarter.libs.BaseActivity
+import com.kickstarter.libs.Build
+import com.kickstarter.libs.CurrentUserType
+import com.kickstarter.libs.qualifiers.RequiresActivityViewModel
+import com.kickstarter.libs.utils.Secrets
+import com.kickstarter.models.User
+import com.kickstarter.viewmodels.HelpSettingsViewModel
+import kotlinx.android.synthetic.main.activity_help_settings.*
+import rx.android.schedulers.AndroidSchedulers
+import java.util.*
+
+@RequiresActivityViewModel(HelpSettingsViewModel.ViewModel::class)
+class HelpSettingsActivity : BaseActivity<HelpSettingsViewModel.ViewModel>() {
+
+    private lateinit var build: Build
+    private lateinit var currentUser: CurrentUserType
+
+    private val loggedOut = R.string.Logged_Out
+    private val mailto = R.string.mailto
+    private val supportEmail = R.string.support_email_to_android
+    private val supportEmailBody = R.string.support_email_body
+    private val supportEmailSubject = R.string.support_email_subject
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_help_settings)
+
+        this.build = environment().build()
+        this.currentUser = environment().currentUser()
+
+        contact.setOnClickListener {
+            this.viewModel.contactClicked()
+
+            this.currentUser.observable()
+                    .take(1)
+                    .observeOn(AndroidSchedulers.mainThread())
+                    .subscribe(this::composeContactEmail)
+        }
+
+        cookie_policy.setOnClickListener{
+            startActivityWithSlideLeftTransition(Intent(this, HelpActivity.CookiePolicy::class.java))
+        }
+
+        help_center.setOnClickListener {
+            startActivityWithSlideLeftTransition(Intent(Intent.ACTION_VIEW, Uri.parse(Secrets.HelpCenter.ENDPOINT)))
+        }
+
+        privacy_policy.setOnClickListener {
+            startActivityWithSlideLeftTransition(Intent(this, HelpActivity.Privacy::class.java))
+        }
+
+        terms_of_use.setOnClickListener {
+            startActivityWithSlideLeftTransition(Intent(this, HelpActivity.Terms::class.java))
+        }
+    }
+
+    private fun composeContactEmail(user: User?) {
```

Comment:
Since we know that `user` can't be null do we should remove the `?`. That way the code will be much cleaner.